public/docs/ts/latest/guide/router.jade

```diff
@@ -331,9 +337,11 @@ figure.image-display
   * including a route parameter (passing the Hero `id` while routing to the "Hero Detail")
   * child routes (the *Crisis Center* has its own routes)
   * the `CanActivate` guard (checking route access)
+  * the `CanActivateChild` guard (checking child route access)
   * the `CanDeactivate` guard (ask permission to discard unsaved changes)
   * the `Resolve` guard (pre-fetching route data)
-  * lazy loading feature area modules
+  * lazy loading feature modules
+  * the `CanLoad` guard (check lazy loaded feature module access)
```

Comment:
Should we word this such that it explains that we are checking if we should indeed load the lazy loaded assets? As currently worded, it implies we have access to the module, but not that the files will be retrieved via XHR in the browser. Just a thought